cfme/tests/control/test_actions.py

```diff
@@ -188,20 +121,27 @@ def compliance_condition(appliance):
         expression="fill_tag(VM and Instance.My Company Tags : Service Level, Gold)"
     )
     yield _compliance_condition
-    _compliance_condition.delete()
+    _compliance_condition.delete_if_exists()
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture
 def compliance_policy(vm_name, policy_name, appliance):
     compliance_policy = appliance.collections.policies.create(
         policies.VMCompliancePolicy,
-        "complaince_{}".format(policy_name),
+        "compliance_{}".format(policy_name),
         scope="fill_field(VM and Instance : Name, INCLUDES, {})".format(vm_name)
     )
     return compliance_policy
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture
+def compliance_tag(appliance):
+    category = appliance.collections.categories.instantiate("Service Level")
+    tag = category.collections.tags.instantiate(name="gold", display_name="Gold")
+    return tag
+
```

Comment:
- As I see ```compliance_tag``` is used in only one test case: ```test_action_check_compliance```. Also it is just initialization for tag and you need only ```appliance``` for it.
- So according to me, we can directly add it in test case rather than creating different fixture.

Reply:
I would prefer to leave this as a fixture. Since it is outside the scope of the test, there is no need to place it inside of the test function itself.